template-files/android/AndroidManifest.xml

```diff
@@ -144,7 +136,6 @@
             android:configChanges="keyboard|keyboardHidden|orientation|screenLayout|screenSize|smallestScreenSize|uiMode"
             android:exported="true"
             android:launchMode="singleTask"
-            android:screenOrientation="portrait"
```

Comment:
not sure if we need to keep this in the mobile AndroidManifest.xml

Reply:
It should've been done in the other PR, but I changed the logic to only apply the lock for phones and to allow rotation for tablets in [this PR](https://github.com/expo/expo/pull/37739/files)